ln/lnurl.go

```diff
@@ -58,7 +59,7 @@ func (wrapper *LNURLoptions) AddInvoice(ctx context.Context, lnInvoice *lnrpc.In
 		return nil, err
 	}
 
-	callbackUrl := fmt.Sprintf("%v?amount=%v", lnAddressUrlResJson.Callback, 1000*lnInvoice.Value)
+	callbackUrl := fmt.Sprintf("%s?amount=%d", lnAddressUrlResJson.Callback, 1000*lnInvoice.Value)
```

Comment:
Typically I make this a constant as well: `MSAT_PER_SAT=1000`, just so it's clear what you are doing here with the `1000*`.